inlong-tubemq/tubemq-client-twins/tubemq-client-go/log/config.go

```diff
@@ -39,3 +39,18 @@ var defaultConfig = &OutputConfig{
 	MaxAge:     3,
 	Level:      "warn",
 }
+
+// SetLogLevel set log level
+func SetLogLevel(level string) {
+	defaultConfig.Level = level
+}
+
+// SetLogPath set log path
+func SetLogPath(path string) {
```

Comment:
It is necessary to check that the path allowed to be set is legal, and it may be necessary to check whether there is permission to output the log